README.md

```diff
@@ -111,64 +109,42 @@ export IMAGE_REGISTRY=<image_registry>
 export REGISTRY_NAMESPACE=<container_org>
 export CONTAINER_TAG=example
```

Comment:
shouldn't we use $IMAGE_TAG here as in https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/Makefile#L8?